integrations/anthropic_middleware.py

```
@@ -122,27 +151,50 @@ def create(self, **kwargs):
-                    f"Line {i}: The quick brown fox jumps over the lazy dog."
-                    for i in range(15)
-                ]),
-            },
-            {"role": "assistant", "content": "I understand the context."},
-            {"role": "user", "content": "What is the main subject?"},
-        ],
-    )
 
-    print(f"Response: {response.content[0].text}")
-    print(f"Stats: {client.get_stats()}")
+__all__ = ["SuperCompressAnthropic", "AsyncSuperCompressAnthropic"]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)